gapii/cc/vulkan_inlines.inl

```diff
@@ -280,6 +280,23 @@ void inline CommandListRecreator<std::shared_ptr<RecreateCopyBufferToImageData>>
         t->mSrcBuffer, t->mDstImage, t->mLayout, buffers.size(), buffers.data());
 }
 
+template <>
+void inline CommandListRecreator<
+    std::shared_ptr<RecreateCopyImageToBufferData>>::
+operator()(VkCommandBuffer commandBuf, CallObserver* observer, VulkanSpy* spy,
+           const std::shared_ptr<RecreateCopyImageToBufferData>& t) {
+  if (!spy->Images.count(t->mSrcImage) || !spy->Buffers.count(t->mDstBuffer)) {
+    return;
+  }
+  std::vector<VkBufferImageCopy> regions;
```

Comment:
We should probably consider reserving the space.
regions.reseve(t->mRegions.size())